src/docs/development/platform-integration/add-to-app-android/add-flutter-screen.md

```diff
@@ -0,0 +1,266 @@
+        .build(currentActivity)
+      );
+  }
+});
+```
+
+Replace `"mySpecialEntrypoint"` with the name of your desired Dart entrypoint, and replace
+`"/my_route"` with your desired initial route.
+
+### Step 5: (Optional) Use a cached FlutterEngine
+
+Every `FlutterActivity` creates its own `FlutterEngine` by default. Each `FlutterEngine` has
+a non-trivial "warm-up" time. This means that launching a standard `FlutterActivity` comes 
+with a brief delay before your Flutter experience becomes visible. To avoid this delay, you
+can warm-up a `FlutterEngine` before arriving at your `FlutterActivity` and then you can use
+your pre-warmed `FlutterEngine` instead.
+
+To pre-warm a `FlutterEngine`, find a reasonable location in your app to instantiate and hold
+a `FlutterEngine`. For a simplistic app, storing a `FlutterEngine` in the `Application` class
+might be a reasonable option:
+
+```java
+public class MyApplication extends Application {
+  private FlutterEngine flutterEngine;
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+
+    // Initialize Flutter.
+    FlutterMain.startInitialization(this);
+    FlutterMain.ensureInitializationComplete(this, null);
+
+    // Create and warm-up a FlutterEngine.
+    flutterEngine = new FlutterEngine(this);
+
+    DartExecutor.DartEntrypoint entrypoint = new DartExecutor.DartEntrypoint(
+      getResources().getAssets(),
+      FlutterMain.findAppBundlePath(this),
+      "main"
+    );
+
+    flutterEngine.getDartExecutor().executeDartEntrypoint(entrypoint);
+  }
+
+  @NonNull
+  public FlutterEngine getFlutterEngine() {
+    return flutterEngine;
+  }
+}
+```
+
+With a pre-warmed `FlutterEngine`, you now need to instruct your `FlutterActivity` to use the
+pre-warmed `FlutterEngine` instead of creating a new one. To accomplish this, subclass
+`FlutterActivity` and override the `provideFlutterEngine()` method:
+
+```java
+public class MyFlutterActivity extends FlutterActivity {
+  @Override
+  @NonNull
+  protected FlutterEngine provideFlutterEngine(@NonNull Context context) {
+    return ((MyApplication) context.getApplicationContext()).getFlutterEngine();
+  }
+}
+```
+
+Now, when you launch `MyFlutterActivity`, there will be no delay in the display of Flutter
+content.
+
+{{site.alert.note}}
+  When re-using a cached engine, you assume responsibility for showing the desired content
+  for the given `Activity` that's using the `FlutterEngine`. Strategies for switching content
+  in a shared `FlutterEngine` are beyond the scope of this guide.
+{{site.alert.end}}
+
+## Add a translucent Flutter screen
+
+{% asset development/platform-integration/add-to-app-android/add-flutter-screen/add-single-flutter-screen-transparent_header.png class="mw-100" alt="Add Flutter Screen With Translucency Header" %}
+
+Most fullscreen Flutter experiences are opaque. However, some apps would like to deploy a
+Flutter screen that looks like a modal, e.g., a dialog, bottom sheet, etc. Flutter supports
```

Comment:
e..g. => for example
Rule: Don’t use e.g.:  https://developers.google.com/style/abbreviations#dont-use